src/components/ProposalApplication.tsx

```diff
@@ -15,6 +16,8 @@ export default function ProposalApplication({
   isStudent,
   isSupervisor,
 }: ProposalApplicationProps) {
+  const [isProposalOpen, setIsProposalOpen] = useState<boolean>(true)
```

Comment:
naming: the modal is open or closed. so, isModalOpen would be more fitting. also for the props

Reply:
done